.vscode/settings.json

```diff
@@ -5,6 +5,7 @@
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
   "eslint.rules.customizations": [{ "rule": "*", "severity": "warn" }],
+  "eslint.experimental.useFlatConfig": true,
```

Comment:
looks like the new version is kinda close https://github.com/microsoft/vscode-eslint/releases